S/Sundials/build_tarballs.jl

```diff
@@ -7,80 +7,46 @@ version = v"3.1.1"
 sources = [
     "https://computation.llnl.gov/projects/sundials/download/sundials-3.1.1.tar.gz" =>
     "a24d643d31ed1f31a25b102a1e1759508ce84b1e4739425ad0e18106ab471a24",
-    "http://faculty.cse.tamu.edu/davis/SuiteSparse/SuiteSparse-4.5.3.tar.gz" =>
-    "6199a3a35fbce82b155fd2349cf81d2b7cddaf0dac218c08cb172f9bc143f37a",
-    "./patches",
+    "./bundled",
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-# SuiteSparse for KLU
-cd $WORKSPACE/srcdir/SuiteSparse*/
-
-# Patches for windows build system
-patch -p0 < $WORKSPACE/srcdir/patches/SuiteSparse_windows.patch
-
-for proj in SuiteSparse_config AMD COLAMD BTF KLU; do
-    cd $WORKSPACE/srcdir/SuiteSparse/$proj
-    make -j${nproc} library
-    INSTALL=$WORKSPACE/destdir/ make install
-done
-
-echo "KLU Includes"
-ls $WORKSPACE/destdir/include
-echo "KLU Lib"
-ls $WORKSPACE/destdir/lib
-
-# Now the full Sundials build
 cd $WORKSPACE/srcdir/sundials-*/
 patch -p0 < $WORKSPACE/srcdir/patches/Sundials_windows.patch
 
-# On 64-bit, we need to patch the BLAS libraries to use the Julia name-mangling scheme.
-if [[ ${nbits} == 64 ]]; then
-    patch -p0 < $WORKSPACE/srcdir/patches/Sundials_ilp64.patch
-fi
-
-mkdir build
-cd build
-
 CMAKE_FLAGS="-DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE="${CMAKE_TARGET_TOOLCHAIN}""
 CMAKE_FLAGS="${CMAKE_FLAGS} -DCMAKE_BUILD_TYPE=Release -DEXAMPLES_ENABLE_C=OFF"
 CMAKE_FLAGS="${CMAKE_FLAGS} -DKLU_ENABLE=ON -DKLU_INCLUDE_DIR=\"$prefix/include/\" -DKLU_LIBRARY_DIR=\"$prefix/lib\""
-CMAKE_FLAGS="${CMAKE_FLAGS} -DBLAS_ENABLE=ON"
-CMAKE_FLAGS="${CMAKE_FLAGS} -DENABLE_LAPACK=ON -DLAPACK_LIBRARIES=$LIBBLAS"
+CMAKE_FLAGS="${CMAKE_FLAGS} -DBLAS_ENABLE=ON -DLAPACK_ENABLE=ON"
 
-if [[ ${nbits} == 32 ]]; then
-    echo "***   32-bit BUILD   ***"
-    LIBBLAS="$prefix/lib/libopenblas.so"
-    cmake ${CMAKE_FLAGS} -DBLAS_LIBRARIES=\"$LIBBLAS\" -DSUNDIALS_INDEX_TYPE=int32_t ..
+if [[ ${nbits} == 64 ]] && [[ ${target} != aarch64* ]]; then
+    patch -p0 < $WORKSPACE/srcdir/patches/Sundials_ilp64.patch
+    CMAKE_FLAGS="${CMAKE_FLAGS} -DBLAS_LIBRARIES=\"-L$prefix/lib -lopenblas64_\" -DLAPACK_LIBRARIES=\"-L$prefix/lib -lopenblas64_\""
```

Comment:
The issue seems to be quote-escaping; I'm going to try changing `\"` to `\\\"` instead.